src/components/modals/ManagePlanConstraintsModal.svelte

```diff
@@ -0,0 +1,333 @@
+        suppressSizeToFit: true,
+        valueGetter: (params: ValueGetterParams<ConstraintMetadata>) => {
+          const { data } = params;
+          if (data) {
+            return !!selectedConstraints[data.id];
+          }
+          return false;
+        },
+        width: 35,
+      },
+    ];
+  }
+  $: if (selectedConstraints) {
+    dataGrid?.redrawRows();
+  }
+
+  function viewConstraint({ id }: Pick<ConstraintMetadata, 'id'>) {
+    const constraint = $constraints.find(c => c.id === id);
+    window.open(
+      `${base}/constraints/edit/${constraint?.id}?${SearchParameters.REVISION}=${
+        constraint?.versions[constraint?.versions.length - 1].revision
+      }&${SearchParameters.MODEL_ID}=${$plan?.model.id}`,
+    );
+  }
+
+  function onToggleConstraint(event: CustomEvent<CellEditingStoppedEvent<ConstraintMetadata, boolean>>) {
+    const {
+      detail: { data, newValue },
+    } = event;
+
+    if (data) {
+      selectedConstraints = {
+        ...selectedConstraints,
+        [data.id]: newValue,
+      };
+    }
+  }
+
+  async function onUpdateConstraints(selectedConstraints: Record<number, boolean>) {
+    if ($plan) {
+      const constraintPlanSpecUpdates: {
+        constraintPlanSpecIdsToDelete: number[];
+        constraintPlanSpecsToAdd: ConstraintPlanSpecInsertInput[];
+      } = Object.keys(selectedConstraints).reduce(
+        (
+          prevConstraintPlanSpecUpdates: {
+            constraintPlanSpecIdsToDelete: number[];
+            constraintPlanSpecsToAdd: ConstraintPlanSpecInsertInput[];
+          },
+          selectedConstraintId: string,
+        ) => {
+          const constraintId = parseInt(selectedConstraintId);
+          const isSelected = selectedConstraints[constraintId];
+
+          if (isSelected) {
+            return {
+              ...prevConstraintPlanSpecUpdates,
+              constraintPlanSpecsToAdd: [
+                ...prevConstraintPlanSpecUpdates.constraintPlanSpecsToAdd,
+                {
+                  constraint_id: constraintId,
+                  constraint_revision: null,
+                  enabled: true,
+                  plan_id: $planId,
+                } as ConstraintPlanSpecInsertInput,
+              ],
+            };
+          } else {
+            return {
+              ...prevConstraintPlanSpecUpdates,
+              constraintPlanSpecIdsToDelete: [
+                ...prevConstraintPlanSpecUpdates.constraintPlanSpecIdsToDelete,
+                constraintId,
+              ],
+            };
+          }
+        },
+        {
+          constraintPlanSpecIdsToDelete: [],
+          constraintPlanSpecsToAdd: [],
+        },
+      );
+
+      await effects.updateConstraintPlanSpecifications(
+        $plan,
+        constraintPlanSpecUpdates.constraintPlanSpecsToAdd,
+        constraintPlanSpecUpdates.constraintPlanSpecIdsToDelete,
+        user,
+      );
+      dispatch('close');
+    }
+  }
+</script>
+
+<Modal height={500} width={600}>
```

Comment:
Could this modal be wider? Tags and this last column (not sure what the "f..." is) are getting cut off 
![image](https://github.com/NASA-AMMOS/aerie-ui/assets/4419607/d4d5838f-fe15-4605-931b-99fcb40465ce)

Reply:
Hold on - didn't re-install...

Reply:
Seeing the checkbox now but think a wider modal would let the tags display not have to be so small?